src/engine/renderer/tr_light.cpp

```diff
@@ -1622,34 +1568,6 @@ void R_ComputeFinalAttenuation( shaderStage_t *pStage, trRefLight_t *light )
 =================
 R_CullLightTriangle
```

Comment:
Leftover comment.

Reply:
It wasn't me. The banner comment for R_CullLightWorldBounds already says "R_CullLightTriangle"